docs/docs/writing-plugins/the-rules-api/processes.mdx

```diff
@@ -7,27 +7,26 @@ How to safely run subprocesses in your plugin.
 
 ---
 
-It is not safe to use `subprocess.run()` like you normally would because this can break caching and will not leverage Pants's parallelism. Instead, Pants has safe alternatives with `Process` and `InteractiveProcess`.
+It is not safe to use `subprocess.run()` like you normally would, because this can break caching and will not leverage Pants's concurrency mechanisms. Instead, Pants has the safe alternatives  `Process` and `InteractiveProcess`.
```

Comment:
```suggestion
It is not safe to use `subprocess.run()` like you normally would, because this can break caching and will not leverage Pants's concurrency mechanisms. Instead, Pants has the safe alternatives `Process` and `InteractiveProcess`.
```